src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -12421,10 +12383,8 @@ void VRFTerminalUnitEquipment::ControlVRF_FluidTCtrl(EnergyPlusData &state,
     Real64 FullOutput;     // unit full output when compressor is operating [W]
     Real64 TempOutput;     // unit output when iteration limit exceeded [W]
     Real64 NoCompOutput;   // output when no active compressor [W]
-    int SolFla;            // Flag of RegulaFalsi solver
     Real64 TempMinPLR;     // min PLR used in Regula Falsi call
     Real64 TempMaxPLR;     // max PLR used in Regula Falsi call
-    bool ContinueIter;     // used when convergence is an issue
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:12424]:(style),[variableScope],The scope of the variable 'SolFla' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:12427]:(style),[variableScope],The scope of the variable 'ContinueIter' can be reduced.